src/sass/app/websocket_pages.scss

```diff
@@ -110,6 +110,10 @@
             height: 25px;
         }
     }
+    ul.hint-info {
+        margin-top: 0;
+        margin-bottom: 0;
```

Comment:
can we use the `no-margin` class?

Reply:
We only want to remove the `margin-top` and `margin-bottom` here and keep the left and right margin.